examples/jenkins-master/jenkins-master-template.json

```diff
@@ -0,0 +1,251 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins-master",
+    "annotations": {
+      "description": "Jenkins service with Kubernetes plugin, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```

Comment:
yeah i understand what it does.  can't we have a simpler sample job that doesn't depend on the git plugin, and just include that job in the image (or just have the tutorial describe to people how to create the job)?